.tekton/scanner-db-pull-request.yaml

```diff
@@ -271,6 +271,8 @@ spec:
             value: $(params.image-expires-after)
           - name: COMMIT_SHA
             value: $(tasks.clone-repository.results.commit)
+          - name: TARGET_STAGE
+            value: scanner-db
```

Comment:
Technically, this isn't necessary since Dockerfile will be built until the final stage by default. Although I'm happy if you keep this for explicitness.